### PR TITLE
update comment

### PR DESCRIPTION
**Pull request by Jiangxinz**
update comment

### CODE REVIEW

File: webservice/cesar/get_log.py [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+    try :
+        res = session.send( req ).json()
+    except Exception as e:
+        print( 'Error: %s' % e )
+    else:
+        stage_build_beans = res[ 'pipelineBuildBean' ][ 'stageBuildBeans' ]
+        print( stage_build_beans )
+        xly = xlyHandler()
+        for stage in stage_build_beans:
+            if stage[ 'stageName' ] in [ 'clone code' ]:
```
[the remark this answers is not in the document]
Comment (Jiangxinz) [reply webservice/cesar/get_log.py]: done

File: webservice/cesar/get_log.py [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+                continue
+            job_group_build_beans = stage[ 'jobGroupBuildBeans' ][ 0 ]
+            for job in job_group_build_beans:
+                if job[ 'jobName' ] in [ 'build-docker-image' ]:
+                    continue
+                job_name = job[ 'jobName' ]
+                status   = job[ 'status' ]
+                if status == 'FAIL':
```
[the remark this answers is not in the document]
Comment (Jiangxinz) [reply webservice/cesar/get_log.py]: done

File: webservice/cesar/get_log.py [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+    return line
+    
+
+def process_failed_log( failed_log_path ):
+    content = []
+    q = []
+    index = 0
+    bias = 10
+    skip_word = [ '+ ', '- ' ]
+    with open( failed_log_path, 'rt' ) as fd:
```
[the remark this answers is not in the document]
Comment (Jiangxinz) [reply webservice/cesar/get_log.py]: done

File: webservice/cesar/get_log.py [hunk 4]
```diff
@@ -0,0 +1,523 @@
...
+from utils.readConfig import ReadConfig
+from utils.auth_ipipe import Get_ipipe_auth
+from utils.handler import xlyHandler
+import heapq
+import re
+
+# target_url = 'https://xly.bce.baidu.com/paddlepaddle/paddle/newipipe/detail/3246472/job/607918://xly.bce.baidu.com/paddlepaddle/Paddle-Bot/newipipe/detail/3222850/job/5980154'
+local_config = ReadConfig( path='conf/config.ini' )
+
+error_patterns = {
```
[the remark this answers is not in the document]
Comment (Jiangxinz) [reply webservice/cesar/get_log.py]: done

**Review by lelelelelez — APPROVED** [review]
LGTM

**Pull request merged**